src/components/ProductPreview/index.js

```diff
@@ -0,0 +1,29 @@
+import Image from 'next/image'
+import { useState } from 'react'
+import { PreviewContainer, Thumbnail, PreviewImage, ThumbnailWrapper } from './ProductPreview.styled'
+
+
+
+const ProductPreview = ({ images }) => {
+  const [selectedImage, setSelectedImage] = useState(images[0])
+
+  const handleThumbnailClick = (image) => {
+    setSelectedImage(image)
+  }
+
+  return (
+    <PreviewContainer>
+      <PreviewImage src={selectedImage} width={300} height={300} />
+      <ThumbnailWrapper>
+      {images.map((image) => (
+        <Thumbnail key={image.id} onClick={() => handleThumbnailClick(image)}>
+          <Image src={image} width={80} height={80} alt={name} />
+        </Thumbnail>
+        
+      ))}
```

Comment:
As far as I see it correctly, `images` is an array of strings, namely the `image paths`.  🧐
For this reason there is
-  no `alt` (because the images arraythe key `name` does not exist)
-  no **key** (because the images array has no key id).
**If you look into your browser console, you will notice it**

Reply:
without the key is no context to the product itself on the product detailpage 😵‍💫